File: ShuffleString.cs
//1528. Shuffle String - Leetcode Easy

public class Solution {
    public string RestoreString(string s, int[] indices) {
        char[] charArrayS = s.ToCharArray();
        
        for ( int i = 0; i < indices.Length; i++ ) {
            charArrayS[indices[i]] = s[i];
        }
        
        string newString = new string(charArrayS);
        
        return newString;
    }
}

File: ShuffleString.swift
//1528. Shuffle String - Leetcode Easy

class Solution {
    func restoreString(_ s: String, _ indices: [Int]) -> String {
        var sArray = Array(s) //turns given string into char array
        
        for i in 0..<indices.count {
            sArray[indices[i]] = Array(s)[i]
        }
        
        return String(sArray)
    }
}
